### DISCUSSION TITLE
在 Vim 里为 Markdown 文档展示导航窗格 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#227] [Announcements]
https://mazhuang.org/2016/08/03/add-outline-for-markdown-in-vim/

**Comment by 3dot141** [comment]
最近刚开始学习vim
找不到 
.ctags  文件
这是个什么后缀的文件，
放到哪里，可以具体说一下吗？

**Comment by mzlogin** [comment]
@Cupid13 
Linux / macOS 下是 `~/.ctags` 文件，Windows 下是 `C:\Users\<username>\.ctags` 文件，没有就新建一个，这个在文中有提到哈。这个文件名字就叫 .ctags。

**Comment by 3dot141** [comment]
谢谢~
我原来是不知道怎么创建
.ctags 文件
然后用
在cmd用
type nul>.ctags
尝试了一下成功了。
然后就可以了。
谢谢~

**Comment by mzlogin** [comment]
@Cupid13 
不客气哈~

**Comment by chenyongze** [comment]
good
